dev/integration_tests/flutter_gallery/lib/demo/material/text_form_field_demo.dart

```diff
@@ -143,10 +143,9 @@ class TextFormFieldDemoState extends State<TextFormFieldDemo> {
     return null;
   }
 
-  Future<bool> _warnUserAboutInvalidData() async {
-    final FormState? form = _formKey.currentState;
-    if (form == null || !_formWasEdited || form.validate()) {
-      return true;
+  Future<void> _onPopped(bool success) async {
```

Comment:
This should call _handlePopped as per style guide

Reply:
You're right ([relevant part of the styleguide](https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo#naming-rules-for-typedefs-and-function-variables)).  I've probably made this mistake elsewhere, I'll clean those up too.